src/System.Private.ServiceModel/tests/Scenarios/Binding/Http/BasicHttpBindingTests.cs

```diff
@@ -4,17 +4,20 @@
 using System;
 using System.ServiceModel;
 using System.ServiceModel.Channels;
+using System.ServiceModel.Tests.Common;
 using System.Text;
 using System.Threading;
 using System.Threading.Tasks;
 using TestTypes;
 using Xunit;
 
-public static class Binding_Http_BasicHttpBindingTests
+public class Binding_Http_BasicHttpBindingTests : OuterLoopTests
```

Comment:
The other option would with a TestFixture which is the xunits way of initializing stuff and its cleaner since its an interface and so we can have multiple if needed -  E.g. https://github.com/SajayAntony/bifrost/blob/master/src/tests/Bridge.Tests/ResourceControllerTests.cs#L15